### ISSUE TITLE
run (jobs -s): do not try to query deleted resources

### ISSUE THREAD

**Issue by yarikoptic** [ReproNim/reproman#360]
Just for our record
```
(git-annex)hopa:~/proj/repronim/eab-run-scripts/SomeGiraffeExample-2[master]git
$> niceman jobs -s
2019-01-12 14:42:57,109 [ERROR  ] Resource i-018c60511ef4a3bf7 (my-nitrc) no longer exists 
2019-01-12 14:42:57,110 [ERROR  ] 'NoneType' object has no attribute 'status' [jobs.py:show_oneline:95] (AttributeError) 

# niceman:
$> git describe
v0.1.0-118-g70eea45
```

note: in general, I feel like any loop going over resources/jobs should be try/except guarded so we do not fail to at least list the Ok resources/jobs

**Comment by kyleam** [comment]
Thanks for catching that.  I'll note this as something to be fixed in #331.

**Comment by yarikoptic** [comment]
Thanks! BTW, upon audience request, I had a not totally unsuccessful live demonstration of the niceman run  ;-) thanks
